src/main/java/com/programmers/app/console/ConsoleReader.java

```diff
@@ -0,0 +1,32 @@
+package com.programmers.app.console;
+
+import java.util.Scanner;
+
+public class ConsoleReader {
+
+    private final Scanner sc;
+
+    public ConsoleReader() {
+        sc = new Scanner(System.in);
+    }
+
+    int readInt() {
+        while (true) {
+            try {
+                return Integer.parseInt(sc.nextLine());
+            } catch (RuntimeException e) {
+                System.out.println("[System] 숫자를 입력해주세요.");
+            }
+        }
+    }
+
+    String readString() {
+        String input = sc.nextLine();
+
+        while (input.equals("")) {
+            input = sc.nextLine();
+        }
```

Comment:
요 부분은 왜 있나요?

Reply:
실수로 아무것도 안치고 엔터 쳤을 때, 입력을 다시 받으려고 넣었습니다

Reply:
오 그렇군요..? 그러면 이런 식으로 수정해봐도 좋을 것 같습니다.

```suggestion
        while (input.strip().isBlank()) {
            input = sc.nextLine();
        }
```

Reply:
의미가 명확해진다고 생각합니다. 감사합니다!